packages/@aws-cdk/aws-appsync/lib/private/mapping-template.ts

```diff
@@ -0,0 +1,113 @@
+/**
```

Comment:
Why is this file called `mapping-template.ts`? There's no such class/function in this file.

Reply:
It holds helper functions for the mapping-templates!

Reply:
we could call it `mapping-templates-utils.ts`?

Reply:
How about something like `conditions.ts`, since those seem to be the classes present here?

Reply:
Okay im good with that